lib/conjur/cli.rb

```diff
@@ -91,6 +94,26 @@ def init!
         load_plugins
         commands_from 'conjur/command'
       end
+
+      def command_version_compatible? command
+        version_compatible = false
+        if command.instance_variable_defined?(:@conjur_min_version)
+          begin
+            appliance_version = Conjur::API.appliance_info["services"]["appliance"]["version"]
+          rescue
+            appliance_version = nil
+          end
+
+          if appliance_version != nil
+            appliance_version = Semantic::Version.new appliance_version
```

Comment:
Requiring the api will require` 'semantic'` and `'semantic/core_ext'` so you can just do `appliance_version.to_version`.